remoteworkitem/trackeritem_repository.go

```diff
@@ -1,10 +1,70 @@
 package remoteworkitem
 
-import "github.com/jinzhu/gorm"
+import (
+	"fmt"
 
-// upload imports the items into database
+	"golang.org/x/net/context"
+
+	"github.com/almighty/almighty-core/app"
+	. "github.com/almighty/almighty-core/criteria"
+	"github.com/almighty/almighty-core/models"
+	"github.com/jinzhu/gorm"
+)
+
+// Upload imports the items into database
 func upload(db *gorm.DB, tqID int, item map[string]string) error {
 	ti := TrackerItem{Item: item["content"], RemoteItemID: item["id"], BatchID: item["batch_id"], TrackerQueryID: uint64(tqID)}
 	err := db.Create(&ti).Error
 	return err
 }
+
+// Map a remote work item into an ALM work item and persist it into the database.
+func convert(ts *models.GormTransactionSupport, tqID int, item map[string]string, provider string) (*app.WorkItem, error) {
```

Comment:
I couldn't find a simpler name for the whole activity. Do you have something more suitable in mind?